CONTRIBUTING.md

```diff
@@ -39,7 +39,7 @@ A typical contributor workflow looks like this:
     - Linting is best handled by your editor for real-time feedback (see
       [Editor integration](https://github.com/palantir/blueprint/wiki/Editor-integration)). Run
       `yarn lint` to be 100% safe.
-    - TypeScript lint errors can often be automatically fixed by TSLint. Run lint fixes with `yarn lint-fix`.
+    - TypeScript lint errors can often be automatically fixed by ESLint. Run lint fixes with `yarn lint-fix`.
```

Comment:
`yarn lint-fix` doesn't match what's in the next file. I would revert the script name change and keep it as `lint-fix`